main.ts

```diff
@@ -81,15 +22,19 @@ async function run(): Promise<void> {
       return
     }
 
-    const architectureToDownload =
-      architecture === 'aarch64' ? 'x86_64' : architecture
+    if (architecture === 'aarch64' && flavor !== 'full') {
+      throw new Error(
+        'On aarch64, only the "full" flavor is supported at this time.'
+      )
+    }
```

Comment:
In order to support other flavors, `please.sh create-sdk-artifact` will need to be updated first to move away from `bitness` first. [Pending final decision](https://github.com/git-for-windows/build-extra/pull/452#discussion_r1053792396) on architecture names that GfW will use.

Reply:
PR to teach `create-sdk-artifact` to use `architecture` instead of `bitness`: https://github.com/git-for-windows/build-extra/pull/462